test/components/operations/Alpha.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Alpha from '../../../src/components/operations/Alpha'
+import renderer from 'react-test-renderer'
+
+test('Alpha does render', () => {
+  const component = renderer.create(
+    <Alpha values={{'mode': 'mask'}} defaults={{
+      'mode': {
+        'default': 'mask',
+        'values': [
+          'mask',
+          'remove',
+          'extract'
+        ]
+      }}} required={[]} />
```

Comment:
could you move the values and defaults to a separate variable? is more readable that way IMO.